### PR TITLE
Add eventlog parse logics for memory monitoring

### PR DESCRIPTION
**Pull request by yawangwang**
Adding eventlog parse logics for memory monitoring.

### CODE REVIEW

**Comment by jkl73** [comment]
We can split the change to 2 PRs, we first merge the eventlog parsing logic, and wait for it to deploy to the service. And then merge the measuring logic PR, so the image test can pass here.

**Comment by yawangwang** [comment]
> We can split the change to 2 PRs, we first merge the eventlog parsing logic, and wait for it to deploy to the service. And then merge the measuring logic PR, so the image test can pass here.

Looks like confidential space VMs failed to start due to unknown COS event type errors returned by `VerifyAttestation` if we add measuring logics to this PR. 
Done.

File: server/eventlog.go [hunk 1]
```diff
@@ -198,6 +199,10 @@ func getVerifiedCosState(coscel cel.CEL) (*pb.AttestedCosState, error) {
 			cosState.Container.OverriddenEnvVars[envName] = envVal
 		case cel.LaunchSeparatorType:
 			seenSeparator = true
+		case cel.MemoryMonitorType:
+			if len(cosTlv.EventContent) > 0 && cosTlv.EventContent[0] == uint8(1) {
```
Comment (jkl73) [inline server/eventlog.go:203]: Can be more definitive here, by making sure the len(cosTlv.EventContent) == 1

Comment (jkl73) [reply server/eventlog.go:203]: And for failsafe, I think the default value for cosState.HealthMonitoring.MemoryEnabled should be true, and when we see the eventContent == 0 here, we can turn it to false

Comment (yawangwang) [reply server/eventlog.go]: > Can be more definitive here, by making sure the len(cosTlv.EventContent) == 1

Done.

Comment (yawangwang) [reply server/eventlog.go]: > And for failsafe, I think the default value for cosState.HealthMonitoring.MemoryEnabled should be true, and when we see the eventContent == 0 here, we can turn it to false

Discussed offline, we agree to keep the default value as false.

**Review by jkl73 — APPROVED** [review]

**Pull request merged**